rust/config/mainnet_config.json

```diff
@@ -131,6 +132,67 @@
       "timelockController": "0x0000000000000000000000000000000000000000",
       "validatorAnnounce": "0x1df063280C4166AF9a725e3828b4dAC6c7113B08"
     },
+    "arthera": {
```

Comment:
Do we want to remove references to arthera for generated files like this one now @tkporter